static/src/js/containers/AuthTokenContainer/AuthTokenContainer.js

```diff
@@ -1,4 +1,4 @@
-import { Component } from 'react'
+import { memo, useEffect } from 'react'
```

Comment:
I think we generally need `React` in scope when using jsx. I'm curious why this hasn't generated any eslint failures. I feel like it should be.

Reply:
`return children`, there isn't any jsx in the document, so we don't need React imported